src/main/java/com/exasol/containers/ExasolContainer.java

```diff
@@ -892,4 +890,15 @@ public String getHostIp() {
     public Optional<X509Certificate> getTlsCertificate() {
         return this.certificateProvider.getCertificate();
     }
+
+    /**
+     * Get the SHA256 fingerprint of the self-signed TLS certificate used by the database in the container for database
```

Comment:
```suggestion
     * Get the SHA256 fingerprint of the TLS certificate used by the database in the container for database
```
This isn't necessarily always self-signed. You can replace it with a proper one.